index.ts

```diff
@@ -286,28 +286,47 @@ TEST: addTests('isQuickPR', [
 	'https://github.com/sindresorhus/refined-github/compare/test-branch?quick_pull=1',
 ]);
 
-const prStateSelector = [
+const stateSelector = [
 	'.State',
 	'[class^="StateLabel"]',
 ].join(',');
 
-export const isDraftPR = (): boolean => $(prStateSelector)!.textContent!.trim() === 'Draft';
+export const isDraftPR = (): boolean => isPR() && $(stateSelector)!.textContent!.trim() === 'Draft';
 export const isOpenPR = (): boolean => {
-	const status = $(prStateSelector)!.textContent!.trim();
+	if (isPR()) {
+		return false;
+	}
+
+	const status = $(stateSelector)!.textContent!.trim();
 	return status === 'Open' || status === 'Draft';
 };
 
 export const isMergedPR = (): boolean => {
-	const status = $(prStateSelector)!.textContent!.trim();
+	if (isPR()) {
+		return false;
+	}
+
+	const status = $(stateSelector)!.textContent!.trim();
 	return status === 'Merged';
```

Comment:
There are no merged issues, so this can be a single line like `return $(sel)?.textContent === 'merged'`

Same for isDraftPR. We should skip the type test where possible for this group of checks

Reply:
Single line